README.md

```diff
@@ -1,46 +1,37 @@
 # `plutarch-quickcheck`
-Public discussion and assistance can be found on [discord](https://discord.gg/yGkjxrYueB) @ #liqwid-plutarch-extra
+Public discussion and assistance can be found on [discord](https://discord.gg/yGkjxrYueB) @ #liqwid-libs
 
 ## What is this?
 
-A helper library to write Plutarch-oriented QuickCheck property tests. It
-includes support for both 'general' QuickCheck-based property testing and testing (some) Plutarch type class laws. These helpers
-integrate seamlessly with
-[`tasty-quickcheck`](https://hackage.haskell.org/package/tasty-quickcheck).
+A helper library to write Plutarch-oriented QuickCheck property tests. 
+Interfaces `plutarch-quickcheck` provide are designed to look and feel
+like writting regular `QuickCheck` properties. 
 
 ## What _exactly_ does this do for me?
 
-QuickCheck is a folkloric minefield and can be quite tedious to use correctly.
-Furthermore, its interactions with Plutarch can be non-obvious, and much
-of the built-in functionality it provides is not very useful for it.
-Furthermore, while Plutarch type classes do have laws, verifying them by hand
-can be complex and error-prone. Lastly, the error output of QuickCheck
-(and, by extension, `tasty-quickcheck`) can be hard to understand.
+`plutarch-quickcheck` *lifts* property written in Plutarch code into 
+`QuickCheck`. For example,
```

Comment:
While this is a lot less text, I feel that we lose a lot of the motivation for this library's existence with this change.